packages/editor/src/store/constants.js

```diff
@@ -7,3 +7,22 @@
 export const EDIT_MERGE_PROPERTIES = new Set( [
 	'meta',
 ] );
+
+/**
+ * Constant for the store module (or reducer) key.
+ * @type {string}
+ */
+export const MODULE_KEY = 'core/editor';
+
+export const POST_UPDATE_TRANSACTION_ID = 'post-update';
+export const SAVE_POST_NOTICE_ID = 'SAVE_POST_NOTICE_ID';
+export const TRASH_POST_NOTICE_ID = 'TRASH_POST_NOTICE_ID';
+export const PERMALINK_POSTNAME_REGEX = /%(?:postname|pagename)%/;
```

Comment:
Where is this referenced? Was there an intention to consolidate constants from the `selectors.js` file here? Because that does not appear to be represented (the constants remain in `selectors.js` and are referenced as defined there).

Reply:
hmm ya, intended to consolidate all constants into one file.  Makes it much easier to reference them (and helps with future refactors).  I'll review/fix.